src/config.rs

```diff
@@ -55,6 +57,7 @@ impl NodeConfig {
             node = node
                 .system_domain(domains::ConfigBuilder::new().core(ConfigBuilder::new().build()));
         }
+        node = node.role(Role::Authority);
```

Comment:
Can you add it to every configuration? This is also true for gemini 3c and devnet